packages/clay-icon/src/ClayIcon.soy

```diff
@@ -6,11 +6,23 @@
 {template .render}
 	{@param spritemap: string}
 	{@param symbol: string}
+	{@param? ariaLabel: string}
```

Comment:
Why can't we use just title as the ariaLabel?

Reply:
Using `title` instead of `ariaLabel` is confusing because if I'm passing a value to the title attribute, I fully expect it to be the SVG's title element.

Reply:
`title` is just that, the title of the icon, is semantically understandable, where it's placed shouldn't care the user IMHO.

Reply:
User's do care. It's extremely frustrating to have an attribute named the same as an element that exists in the svg spec but is applied somewhere else. We even placed it in the title element and now we moving it to aria-label? What if we need to place something in the title element in the future what will we call it then?

Reply:
My point is that this is no ClaySVG, is ClayIcon, user shouldn't care about the markup but that an icon is placed. `title` could refer to the container title attribute too, or `ariaLabel` to an svg attribute.